app/main/templates/series_view.html

```diff
@@ -4,82 +4,97 @@
 {% block title %}{{title}}{% endblock %}
 
 {% block app_content %}
-    <!-- Series description -->
-    <div class="row">
-        <h2 class="text-center">DICOM series overview</h2>
-        <table class="table table-striped"> <!-- Series information -->
-            <tr>
-                <th>Series description</th>
-                <td>{{ series.description }}</td>
-            </tr>
-            <tr>
-                <th>Acquisition date</th>
-                <td>{{ series.series_datetime }}</td>
-            </tr>
-            <tr>
-                <th>Uploaded date</th>
-                <td>{{ series.created_at.format('YYYY-MM-DD HH:mm') }}</td>
-            </tr>
-            <tr>
-                <th>Files in series</th>
-                <td>{{ series.series_files }}</td>
-            </tr>
-            <tr>
-                <th>Reports made</th>
-                <td>{{ series.has_report }}</td>
-            </tr>
-        </table>
-        </br>
+
+    <div class="container py-5">
+        <div class="row">
+            <h1 class="h1 text-center justify-content-center">DICOM Series Overview</h1>
```

Comment:
I don't like the capitalisation of series overview, but it is just personal preference

Reply:
I've just gone for capitalisation consistency across the site.